scipy/stats/_multicomp.py

```diff
@@ -180,7 +180,7 @@ def confidence_interval(
         return self._ci
 
 
-@xp_capabilities(np_only=True)
+@xp_capabilities(out_of_scope=True)
 @_transition_to_rng('random_state', replace_doc=False)
 def dunnett(
```

Comment:
The time-consuming part of this is multivariate t CDF, and that is not being translated to the array API any time soon.